sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> Using both options ..CCA and ..CSMA does not really fit to a practical implementation related to the atmel device, right?

Depends. As I said, the `AUTOCCA` behaviour can be emulated (in HW). And as a SW-CSMA would test for `NETCONF_OPT_CSMA` first and then falling back to `NETCONF_OPT_AUTOCCA` there would be no problem either. But we must ensure that the documentation really makes clear the difference.

> Omitting the ..CCA option would fit to an implementation on the atmel device, but that would be very ambiguous when it comes to the KW2x device since it actually does not support CSMA.

Another argument for having both coexist for the moment.